### DISCUSSION TITLE
theia blueprint browser build problem

### DISCUSSION THREAD

**Discussion by wss29** [eclipse-theia/theia#13816] [Q&A]
hi,
   after I download theia-blueprint 1.50.1, 
   1. I follow the read-me steps to build it, yarn and yarn build, and everything is ok, I can start theia-ide through "yarn electron start" or "yarn start browser"
   2. now I want just to build browser,  it will take less time than building both electron and browser. so I execute "yarn browser build", and the build seems ok, no error appears
   3. I execute "yarn browser start", and I get following error
![image](https://github.com/eclipse-theia/theia/assets/4537809/8a485c35-0e16-4d74-8f92-c4603d1483e3)
 what step did I miss? must I build browser and electron every time? Can not I build browser or electron or both every time I want ?

**Comment by msujew** [comment]
Hey @wss29,

there is probably some stuck cache that doesn't get cleared correctly. Can you try running `git clean -xfd` and then the `yarn install && yarn browser build` commands?

**Reply by wss29** [reply]
Hi @msujew ,
1. running git clean -xfd and yan install && yarn build:extensions && yarn browser  build. it works.
2 In fact I found once you have run yarn electron build or yarn build, after that you run yarn browser build, yarn browser start will get errors.
3. it seems that electron build always cause stuck cache, so when I only need browser I will always never run yarn electron build or yarn build, or I should clean and yarn again
